packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -1,4 +1,4 @@
-import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import { attr, FASTElement, observable, Observable } from "@microsoft/fast-element";
 import { isHTMLElement } from "@microsoft/fast-web-utilities";
```

Comment:
What other dependencies do we have coming from fast-web-utilities. If this is the only one, we could easily remove it since this is a pretty simple check to make inline.

Reply:
For foundation, there are a lot of single-use constants and enums: `Direction`, `Orientation`, `KeyCodes` (as well as individual ones like `KeyCodeEscape`). There are also some functions like `wrapInBounds`, `limit`, `canUseFocusVisible`, and `DOM.queueUpdate`/`DOM.nextUpdate`.

For components, all but one are `SystemColors`, `Orientation`, and `Direction`, which are all enums. The remainder is the `RtlScrollConverter` class, which is only used in the `anchored-region` storybook story.

Reply:
We have quite a few - I don't think removing here will make a significant reduction. If we want to explore dropping the package as a dependency we should probably take it to an issue and determine how we can do so.

Reply:
Sounds good. I wasn't very familiar with what we were using from that package, so I just wanted to make sure, since this particular utility is pretty simple to inline if needed. No reason to change anything.